pandas/core/indexes/range.py

```diff
@@ -48,6 +48,10 @@ class RangeIndex(Int64Index):
     --------
     Index : The base pandas Index type
     Int64Index : Index of int64 data
+
+    Methods
+    -------
+    from_range
```

Comment:
Is it because there is no Attributes section that you get those warnings? 
(but you said they were fixed with numpydoc master? Then the other places that do an empty section with None are also not needed?)